### PR TITLE
Integrate labs-ui

### PR DESCRIPTION
**Pull request by trbmcginnis**
Integrate [labs-ui](https://github.com/NYCPlanning/labs-ui)

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+// Foundation
+@import 'foundation-sites/foundation';
+@include foundation-everything;
+
+// Labs UI modules (import all or some)
+@import 'labs-ui/all-modules';
+
+// Custom app modules and styles
+@import 'layouts/_l-default';
```
Comment (andycochran) [inline app/styles/app.scss]: You're importing a custom app-specific Sass file that doesn't exist.

Comment (trbmcginnis) [reply app/styles/app.scss]: Okay I will remove thanks!

File: config/environment.js [hunk 2]
```diff
@@ -21,6 +21,13 @@ module.exports = function(environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     }
```
Comment (godfreyyeung) [inline config/environment.js]: There needs to be a comma here

Comment (trbmcginnis) [reply config/environment.js]: will add

**Pull request closed without merging**